src/server/troubleshoot/index.md

```diff
@@ -17,3 +17,18 @@ Did you get an error that the server is not properly configured?
    
 2. Restart the container with the `MERGIN_BASE_URL` variable
 
+## Emails are not sent
+
+If you are not receiving emails, check that the following environment variables are set correctly:
+
+* `MAIL_DEFAULT_SENDER` is mandatory and should be a valid email address
+* `MAIL_SERVER` should be a valid url to SMTP server
+* `MAIL_PORT` if you have different port than 587, configure it with this variable
+* `MAIL_SUPPRESS_SEND` check if you set it to `False`
+
+In some deployments are SMTP servers that are not supporting authentication and TLS. In this case, you can disable authentication by not defining variable `MAIL_USERNAME` and `MAIL_PASSWORD` (which are by default `None`). If your SMTP server does not support TLS or SSL you can disable encryption by setting `MAIL_USE_TLS` and `MAIL_USE_SSL` to `false`. It's recommended to use authentication and TLS encryption.
+
+Server is sending emails with celery worker. If you are not receiving emails, check if celery worker is running. You can check logs for sending emails in `celery-worker` container:
```

Comment:
```suggestion
Server is sending emails with a celery worker. If you are not receiving emails, check if celery worker is running. Check logs for sending emails in the `celery-worker` container:
```

Reply:
How about "Emails are sent by celery worker container (link to docker-compose file)."